apps/strava/strava.star

```diff
@@ -34,7 +31,8 @@ PREVIEW_DATA = {
     "elevation_gain": 125800,
 }
 
-CACHE_TTL = 60 * 60 * 24  # updates once daily
+CACHE_TTL = 60 * 60 * 36  # updates once every 36 hours.
```

Comment:
I've updated this to be every 36 hours to help alleviate our API usage until we can get our limit increase request approved.